apps/vaporgui/MainForm.cpp

```diff
@@ -301,6 +301,7 @@ MainForm::MainForm(
     _tabDockWindow = new QDockWidget(this );
     addDockWidget(Qt::LeftDockWidgetArea, _tabDockWindow );
     _tabDockWindow->setAllowedAreas(Qt::LeftDockWidgetArea | Qt::RightDockWidgetArea);
+    _tabDockWindow->setFeatures(QDockWidget::NoDockWidgetFeatures);
```

Comment:
What does do?

Reply:
It fixes  #1976 by disallowing floating the left bar.